Dockerfile

```diff
@@ -21,15 +21,13 @@ RUN cp /src/config.sample.json /src/webapp/config.json
 
 FROM nginxinc/nginx-unprivileged:${DEBIAN_VERSION} AS nginx
 
-COPY --from=builder /src/webapp /app
-
 # Override default nginx config. Templates in `/etc/nginx/templates` are passed
 # through `envsubst` by the nginx docker image entry point.
 COPY /docker/nginx-templates/* /etc/nginx/templates/
 
 USER root
-RUN rm -rf /usr/share/nginx/html \
-  && ln -s /app /usr/share/nginx/html
+RUN rm -rf /usr/share/nginx/html
+COPY --from=builder /src/webapp /usr/share/nginx/html
```

Comment:
This will break any deployments which bind mount to `/app` as we instruct them to https://github.com/element-hq/element-web/blob/develop/docs/install.md#docker

the reason we link is backwards compatibility, it used to be `/usr/share/nginx/html` hence by linking we support both.

Reply:
Ok, rewound this commit